.gitignore

```diff
@@ -1,2 +1 @@
 bazel-*
-go.sum
```

Comment:
Should go.sum be kept in `.gitignore`?

Reply:
No, I was following this [SO question](https://stackoverflow.com/q/53837919) pointing to golang FAQ.

> Typically your module's go.sum file should be committed along with your go.mod file.\
> — https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file